doc/api/webstreams.md

```diff
@@ -1467,6 +1509,49 @@ added: v16.7.0
 * Returns: {Promise} Fulfills with a {Blob} containing the full contents
   of the stream.
 
+```mjs
+import { blob } from 'node:stream/consumers';
+import { Readable } from 'node:stream';
+
+// Creates a 1_000_000 items array of objects
+const items = Array.from(
+  {
+    length: 1e6
+  },
+  () => ({
+    message: 'hello world from consumers!'
+  })
+);
+
+const readable = Readable.from(JSON.stringify(items)); // Creates a readable stream from the array
+// Consumes data stream as a blob: '[{"message": "hello world from consumers!"},{...},{...}]'
+const data = await blob(readable);
+console.log(`from readable: ${data.size}`); // From readable: 42000001
+```
+
+```cjs
+const { blob } = require('node:stream/consumers');
+const { Readable } = require('node:stream');
+
+// Creates a 1_000_000 items array of objects
+const items = Array.from(
+  {
+    length: 1e6
+  },
+  () => ({
+    message: 'hello world from consumers!'
+  })
+);
+
+async function readFromStream() {
```

Comment:
if you'd like to keep functions for cjs, I'd rename them. Eg. instead of `readFromStream` move it to `readBlobStream`